Rakefile

```diff
@@ -194,4 +219,50 @@ end
 def cprint(string)
   print "\033[1m#{string}\033[0m"
 end
+
+def retrieve_vm_as_ova(vmname)
+  ovaname = "puppet-#{VMTYPE}vm-#{PEVERSION}-#{RELEASE}"
+  vcenter_config = File.join(CACHEDIR, ".vmwarecfg.yml") || ENV["VCENTER_CONFIG"]
+  vcenter_settings = YAML::load(File.open(vcenter_config))
+  FileUtils.rm_rf(OVFDIR) if File.directory?(OVFDIR)
+  FileUtils.mkdir_p(OVFDIR)
+  Dir.chdir(BUILDDIR) do
+    verbose(false) do
+      sh(%Q</usr/bin/ovftool --noSSLVerify --targetType=OVA --compress=9 --name=#{ovaname} --powerOffSource vi://#{vcenter_settings['username']}\\@puppetlabs.com:#{vcenter_settings['password']}@vmware-vc2.ops.puppetlabs.net/opdx2/vm/vmpooler/centos-6-i386/#{vmname}  #{OVFDIR}/>)
+    end
+  end
+  File.join(OVFDIR, ovaname) + ".ova"
+end
+
+def ship_vm_to_vmware(vmpath)
+  require 'rbvmomi'
+  vcenter_config = File.join(CACHEDIR, ".vmwarecfg.yml") || ENV["VCENTER_CONFIG"]
+  vcenter_settings = YAML::load(File.open(vcenter_config))
+  Dir.chdir(BUILDDIR) do
+    verbose(false) do
+      sh(%Q</usr/bin/ovftool --noSSLVerify --network='delivery' --datastore='instance1' -o --powerOffTarget -n=#{VMTYPE} #{vmpath} vi://#{vcenter_settings['username']}\@puppetlabs.com:#{vcenter_settings['password']}@vmware-vc2.ops.puppetlabs.net/opdx2/host/mac1>)
+    end
+  end
+  vim = RbVmomi::VIM.connect(
+    :host => 'vmware-vc2.ops.puppetlabs.net', 
+    :user => "#{vcenter_settings['username']}\@puppetlabs.com", 
+    :password => "#{vcenter_settings['password']}", 
+    :insecure => 'true')
+  rootFolder = vim.serviceInstance.content.rootFolder
+  dc = rootFolder.childEntity.grep(RbVmomi::VIM::Datacenter).find { |x| x.name == "opdx2" } or fail "datacenter not found"
+  vm = dc.find_vm(VMTYPE) or fail "VM not found"
+  cputs "Powering on VM"
+  vm.PowerOnVM_Task.wait_for_completion
+  5.times do
```

Comment:
Totally, good catch, I meant to come back and add a check for that and never did.